### PR TITLE
Cache PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by szepeviktor**
It will take much less time!

Was
image

### CODE REVIEW

**Comment by szepeviktor** [comment]
Works
image

**Comment by szepeviktor** [comment]
# 💣 1 second

> Using cache file ".php-cs-fixer.cache".

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
         if: startsWith(matrix.os, 'windows') == false
         run: |
           if ! git diff --exit-code --name-status; then
-            echo "::error file=$(git diff --name-status | head -n1)::PLEASE RUN bin/unipoints build"
+            echo "::error file=$(git diff --name-status | head -n 1)::PLEASE RUN bin/unipoints build"
```
Comment (mlocati) [inline .github/workflows/tests.yml:56]: Mechele's coming from decades of posix scripting, and he's always written short options in the most compact form

Comment (szepeviktor) [reply .github/workflows/tests.yml:56]: NO!! Michele is not so old.
Watch this: `head -1` 😲

Comment (mlocati) [reply .github/workflows/tests.yml:56]: > NO!! Michele is not so old.

Yet not so young (I've been programming for 34 years and counting)

**Pull request merged**